### PR TITLE
Sitton intake to cargo fix

### PR DESCRIPTION
**Pull request by tal-sitton**

### CODE REVIEW

File: src/main/java/frc/robot/Subsystems/Transfer.java [hunk 1]
```diff
@@ -14,7 +14,8 @@
  */
 public class Transfer extends MotoredGenericSubsystem {
 
-    private final Supplier<Double> speed = rootNamespace.addConstantDouble("speed", -0.5);
+    private final Supplier<Double> MOVE_SPEED = rootNamespace.addConstantDouble("move speed", -0.5);
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/Subsystems/Transfer.java:17]: use public

File: src/main/java/frc/robot/Subsystems/Transfer.java [hunk 2]
```diff
@@ -14,7 +14,8 @@
...
+    private final Supplier<Double> RELEASE_SPEED = rootNamespace.addConstantDouble("release speed", -0.7);
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/Subsystems/Transfer.java:18]: ditto

File: src/main/java/frc/robot/Subsystems/Transfer.java [hunk 3]
```diff
@@ -53,12 +54,16 @@ public double getTransferMoveTimeout() {
...
+    }
+
+    public Supplier<Double> getTransferReleaseSpeed() {
+        return RELEASE_SPEED;
     }
 
     @Override
     public void configureDashboard() {
-        rootNamespace.putData("transfer", new MoveGenericSubsystem(this, speed));
+        rootNamespace.putData("transfer", new MoveGenericSubsystem(this, MOVE_SPEED.get()));
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/Subsystems/Transfer.java:67]: why are you using `get()`?

File: src/main/java/frc/robot/Commands/IntakeCargo.java [hunk 4]
```diff
@@ -36,15 +35,21 @@ public boolean isFinished() {
                         new MoveGenericSubsystem(intakeToTransfer, IntakeToTransfer.SPEED) {
                             @Override
                             public boolean isFinished() {
-                                return transfer.getEntranceSensor();
+                                return (transfer.getEntranceSensor() && !hasCargo) || (intakeToTransfer.getLimit() && hasCargo);
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/Commands/IntakeCargo.java:38]: first check if it has cargo and then check the limit

**Review by Yoel-Perman-Brilliant — DISMISSED** [review]

File: src/main/java/frc/robot/Commands/IntakeCargo.java [hunk 5]
```diff
@@ -35,7 +35,7 @@ public boolean isFinished() {
                         new MoveGenericSubsystem(intakeToTransfer, IntakeToTransfer.SPEED) {
                             @Override
                             public boolean isFinished() {
-                                return (transfer.getEntranceSensor() && !hasCargo) || (intakeToTransfer.getLimit() && hasCargo);
+                                return (!hasCargo && transfer.getEntranceSensor()) || (!hasCargo && intakeToTransfer.getLimit());
```
Comment (OfriRosenbaum) [inline src/main/java/frc/robot/Commands/IntakeCargo.java:38]: no

**Review by OfriRosenbaum — CHANGES_REQUESTED** [review]

**Review by OfriRosenbaum — APPROVED** [review]

**Pull request merged**